src/systems/ec/specs/components/media-container/demo/data--video.js

```diff
@@ -0,0 +1,33 @@
+// Simple content for demo
+module.exports = {
+  alt:
+    'The European Commission has put forward ambitious yet realistic proposals for a modern EU budget. It is time for an EU budget that reflects rapid developments in innovation, the economy, the environment and geopolitics, amongst others.',
+  image:
+    'https://ec.europa.eu/education/sites/education/files/jean-monnet-gs-banner.jpg',
+  description:
+    'The European Commission has put forward ambitious yet realistic proposals for a modern EU budget. It is time for an EU budget that reflects rapid developments in innovation, the economy, the environment and geopolitics, amongst others.',
+  sources: [
+    {
+      src: 'http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4',
```

Comment:
can you use the `https` version instead? in order to avoid `Mixed Content: The page at 'https://deploy-preview-823--europa-component-library.netlify.com/storybook/ec/index.html?selectedKind=MediaContainer&selectedStory=video&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs' was loaded over HTTPS, but requested an insecure video 'http://clips.vorwaerts-gmbh.de/big_buck_bunny.webm'. This content should also be served over HTTPS.`